api/client/cli.go

```diff
@@ -93,10 +93,11 @@ func (cli *DockerCli) Subcmd(name, signature, description string, exitOnError bo
 	flags := flag.NewFlagSet(name, errorHandling)
 	flags.Usage = func() {
 		options := ""
+		tempSignature := " " + signature
```

Comment:
since we're assuming 'signature' will never be blank you could remove this and just add the space to the printf.
But if we don't want to assume it will always have a value then you'd need an if-statement around this.